cmd/DAGInfo.go

```diff
@@ -0,0 +1,159 @@
+package cmd
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	// "io/ioutil"
+	// "path/filepath"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/qri-io/dag"
+	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/lib"
+	"github.com/spf13/cobra"
+)
+
+// NewDAGInfoCommand creates a new `qri daginfo` command that generates a daginfo for a given
+// dataset reference. Referenced dataset must be stored in local CAFS
+func NewDAGInfoCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
```

Comment:
I was going to ask for tests on this, but I think we should first consider merging `qri manifest` and `qri daginfo` into one master `qri dag` command. so:

`qri dag info`
`qri dag manifest`

So I think we should merge this & follow up with a quick PR that does that, and test _that_ command instead

Reply:
I really like this!